docs/_articles/en/index.md

```diff
@@ -4,16 +4,28 @@ lang: en
 permalink: /
 ---
 
-This extension pack includes tools for developing on the Salesforce platform in the lightweight, extensible VS Code editor. These tools provide features for working with development orgs (scratch orgs, sandboxes, and DE orgs), Apex, Aura components, and Visualforce.
+## One Set of Extensions, Two Ways to Code
+
+You can install the Salesforce Extensions in VS Code on the desktop or access them via Code Builder in your browser.
+
+The Salesforce Extension pack includes tools for developing on the Salesforce platform in the lightweight, extensible VS Code editor. These tools provide features for working with development orgs (scratch orgs, sandboxes, and DE orgs), Apex, Aura components, and Visualforce.
+
+### Salesforce Extensions for Desktop
+
+Want to develop locally? Already using VS Code? Install Salesforce Extensions and Salesforce CLI to develop quickly and more productively on Salesforce platform.
 
 <span>Download using the links below, or read the [installation instructions](./en/getting-started/install).<span><br/><a class="slds-button slds-button_neutral landing__header-cta slds-m-vertical--x-large" href="https://code.visualstudio.com">Download Visual Studio Code</a><a class="slds-button slds-button_brand landing__header-cta slds-m-vertical--x-large" href="https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-vscode">Install the Salesforce Extensions</a>
 
 ![GIF showing Apex code completion, pushing source to a scratch org, and running Apex tests](./images/overview.gif)
 
+### Code Builder (Beta)
+
+Code Builder is a browser-based version of Salesforce Extensions, with everything installed and pre-configured for you. It's provides all the goodness of the desktop experience, but provides you the flexibility to work anywhere, from any computer. To participate in the beta, &&& DO THIS (or we'll cross ref to instructions/article)
```

Comment:
@ekapner - Perhaps for next week's release, we just point to the [announcement blog](https://developer.salesforce.com/blogs/2020/06/introducing-code-builder.html) and say something like "Check back soon for details about how to participate in the beta!"  We can then update with beta details at launch